src/components/YourCompany/FolioNumber.vue

```diff
@@ -1,89 +1,262 @@
 <template>
-  <div>
-    <v-row no-gutters v-if="!isEditing" id="folio-number-read-only">
+  <div id="folio-number">
```

Comment:
This component also looks a lot like the Business Contact Info component, intentionally!

Reply:
Does it also update Auth in realtime, like contact info and whatnot?

Reply:
Yes! (Which is why I implemented error handling for both.)

Reply:
awesome! 👍